celements-model/src/main/java/com/celements/query/IQueryExecutionServiceRole.java

```diff
@@ -33,6 +33,7 @@ public int executeWriteHQL(String hql, Map<String, Object> binds, WikiReference
 
   public List<DocumentReference> executeAndGetDocRefs(Query query) throws QueryException;
 
-  public boolean existsIndex(String database, String table, String name) throws XWikiException;
+  public boolean existsIndex(@NotNull WikiReference wikiRef, @NotNull String table,
```

Comment:
directly changed signature without deprecation since this is method is new in 3.0 anyway